fern/docs/pages/installation/installation.mdx

```diff
@@ -206,6 +174,11 @@ You might encounter several issues:
   Most likely you are missing some dev tools in your machine (updated C++ compiler, CUDA is not on PATH, etc.).
   If you encounter any of these issues, please open an issue and we'll try to help.
 
+One of the first reflex to adopt is: get more information.
+If, during your installation, something does not go as planned, retry in *verbose* mode, and see what goes wrong.
+
+For example, when installing packages with `pip install`, you can add the option `-vvv` to show the details of the installation.
+
 #### Troubleshooting: C++ Compiler
```

Comment:
Where it says 

** For OSX **

Should be 

**For OSX**

Otherwise formatting is broken

Reply:
(no space between asterisks and characters)